src/lib/es2015.core.d.ts

```diff
@@ -298,6 +298,12 @@ interface ObjectConstructor {
      */
     getOwnPropertySymbols(o: any): symbol[];
 
+    /**
+      * Returns the names of the enumerable properties and methods of an object.
+      * @param o Object that contains the properties and methods. This can be an object that you created or an existing Document Object Model (DOM) object.
+      */
+    keys(o: {}): string[];

```

Comment:
I have a question, should these two be something like:

```
keys<T = {}>(o: T): (keyof T)[] {
```

Reply:
Someone finally explained what this means :-) thanks, I'll update both the es5 and es2015 `keys` signatures to preserve the type of the keys of the input object.